lib/widgets/message_list.dart

```diff
@@ -964,10 +963,10 @@ class MessageWithPossibleSender extends StatelessWidget {
           if (senderRow != null)
             Padding(padding: const EdgeInsets.fromLTRB(16, 2, 16, 0),
               child: senderRow),
-          Row(crossAxisAlignment: CrossAxisAlignment.start, children: [
```

Comment:
msglist: Fix star icon alignment issue.
    
    By aligning the row using `textBaseline`, we no longer need to apply
    the padding to the star.

Neat!